src/worker/exporters/nvidia_exporter.py

```diff
@@ -169,7 +171,18 @@ def InitGauges(self):
                 ],
             )
 
+    def InitcounterConfig(self):
```

Comment:
A bit nitpicky, but can should the "counter" part be capitalized?

Reply:
done